docs/process.md

```diff
@@ -41,105 +42,65 @@ Costs are
-
-See [`perlgit.pod`](https://github.com/Perl/perl5/blob/blead/pod/perlgit.pod#on-merging-and-rebasing) for how to do this with `git merge --no-ff`. This **has** to be done at the command-line - this style of merging does **not** correspond to any of the "merge" options in the GitHub UI.
-
-There is an ongoing discussion on how we decide whether to merge a PR, with the latest proposal being:
-
-* After an appropriate period, if there are not strong disagreements, and the PSC haven't rejected it, a committer will merge the PR.
-* Trivial or obviously-correct changes may be committed directly. I.e., the appropriate length is sometimes zero.
-* If objections are raised, they need to be addressed (meaning "clearly replied to", and for an RFC the reasoning recorded in "Rejected Ideas")
-* If the objection is from a subject matter expert, you need to come to an agreement.
-* If the PR stalls, ping the PSC for adjudication. We expect this will be quite rare either you should have gone through the proposal process, or it's straightforward.
-* PRs from non-committers are expected to have more scrutiny.
-
-What's an appropriate period?
-
-* For "I think this is right but want to give a chance for comments", a couple of days is probably fine.
-* For "this is a significant change that I could really use feedback on," a week or more is probably best, and the PR should probably be flagged to the list as wanting attention.
-* The later in the release cycle, the stricter we should be with ourselves.
-
-Don't panic: changes can be reverted.
+## The Process
 
+![a flowchart of the process described below](/images/flowchart.png)
+
+
+### Pre-RFC
+
+The RFC process starts with an formal proposal to add or change a language feature in Perl.  But you, the prospective author of an RFC, shouldn't start by writing one.  Start by posting to p5p that you have an idea.  Explain what problem you're solving, how you think you can solve it, and what prior art you looked at.  Be clear and concise.  Make it easy for the rest of the list to see what you're suggesting without reading an enormous wall of text, but don't lose so much detail as to be meaningless.
+
+You are taking the temperature of the list.  If there is a great outcry that this is a bad idea, or has been tried before, or was explicitly rejected before, you should probably stop.  If everyone says, "Sounds great, flesh it out!", or if you're not sure, you should post, requesting that the PSC approve producing a draft.  If they do, move on to "Draft Proposal" below.  If not, they'll provide more feedback like "this is definitely impossible" or "you need to provide more information" or so on.
```

Comment:
The "If there is a great outcry" sentence led me to think that this paragraph would be in three parts:
* If there's a great outcry, give up
* If there's great acclaim, proceed
* Otherwise, proceed with caution

But that's not what you're saying; you're saying that if everybody says "this is terrible", give up, and otherwise move on to the next stage. So I think the word "otherwise" should be involved here; or you should split this paragraph into two, e.g.

> You are taking the temperature of the list. If there is a great outcry that this is a bad idea, or has been tried before, or was explicitly rejected before, you should probably stop. No hard feelings!
>
> If your idea isn't considered to be obviously bad, you're ready to move on to the next step. You should post a follow-up, requesting that the PSC approve producing a draft. *continue as before*